guardian/gateway
# chore(cloudformation): Only allow IMDSv2 on Gateway instances

## What does this change?

As per https://docs.aws.amazon.com/securityhub/latest/userguide/ec2-controls.html#ec2-8 we should not be allowing access to IMDSv1 anymore.

This resolves 4 HIGH FSBP vulnerabilities.

## How to test

Check Cloudwatch metrics for any usage of `MetadataNoToken`, if no usage then it is safe for us to disable IMDSv1.

<img width="1681" alt="image" src="https://github.com/user-attachments/assets/6d8d900d-af67-4e39-bb9d-131cbb3a04a4">
